index.html

```diff
@@ -21,6 +21,10 @@ <h2>Store</h2>
 			</a>
 			<div class="news">
 				<h2>Latest News</h2>
+				<article>
+					<h3><a href="https://kafka-summit.org/events/kafka-summit-london-2020/" target="_blank" rel="noopener noreferrer">Kafka Summit London</a></h3>
+					<time>Apr 27 - Apr 28, 2020</time>
+				</article>
```

Comment:
LGTM -- seems we should remove the KS SF entry? It's not relevant any longer.

Reply:
Agreed. Removed....